README.md

```diff
@@ -35,9 +35,9 @@ and locate the relevant ID.
 
 This will do the following:
 
-- Delete all upcoming support rota bookings in Productive,
-- Fetch all support rota info from the API, importing them into Productive as a booking against the Support project
-  for the appropriate person.
+This will fetch all leave bookings from Productive and the Support Rota, deleting any that are
```

Comment:
I'm not sure what leave bookings means in the context of support rota, did you mean to put this?

Reply:
Oops, that's a copypasta error from the BreatheToProductive README 🤦